ui/app/components/dataset/DatasetFormField.tsx

```diff
@@ -1,6 +1,6 @@
 import type { Control, FieldPath, FieldValues } from "react-hook-form";
 import { FormField, FormItem, FormLabel } from "~/components/ui/form";
-import { DatasetSelector } from "~/components/dataset/DatasetSelector";
+import DatasetSelector from "~/components/dataset/DatasetSelector";
```

Comment:
Just my opinion here, but I tend to recommend folks use named exports over defaults as a rule. Named exports are generally more searchable since they are guaranteed to have a reference to the name, whereas default exports can be renamed to anything when imported. A few other reasons in some of the references below.

Not a hill to die on by any means, just my perspective! And one shared by a few others! [[1](https://x.com/mjackson/status/1602837700590989312), [2](https://www.olivare.net/blog/2023/death-to-default-exports), [3](https://webdeveloper.beehiiv.com/p/avoid-default-exports-javascript-modules), [4](https://dev.to/phuocng/avoid-using-default-exports-a1c#:~:text=Default%20exports%20can%20be%20confusing,find%20the%20functions%20they%20need.)]